packages/react-instantsearch-dom/src/components/__tests__/InfiniteHits.js

```diff
@@ -46,27 +50,108 @@ describe('InfiniteHits', () => {
     const hits = [{ objectID: 0 }, { objectID: 1 }, { objectID: 2 }];
     const wrapped = mount(
       <InfiniteHits
-        refine={mockedRefine}
+        refinePrevious={() => undefined}
+        refineNext={mockedRefine}
         hitComponent={Hit}
         hits={hits}
         hasMore={true}
+        hasPrevious={false}
       />
     );
     expect(mockedRefine.mock.calls).toHaveLength(0);
     wrapped.find('button').simulate('click');
     expect(mockedRefine.mock.calls).toHaveLength(1);
   });
 
-  it('Button is disabled when it is the last page', () => {
+  it('calls refineNext when the load more button is clicked', () => {
```

Comment:
From what I've understood this test is exactly the same than the one above. 

We can get rid of the previous one since `refine` is not provided anymore.